recipes/annembed/meta.yaml

```
@@ -20,6 +19,9 @@ requirements:
     - {{ compiler('rust') }}
     - make
     - cmake
+    - openblas
```

I see that `openblas` feature is used only for Linux in the build.sh file.
Maybe you want to scope this dependency ?!
```suggestion
    - openblas  # [linux]
```